api/desecapi/metrics.py

```diff
@@ -74,6 +74,19 @@ def set_histogram(name, *args, **kwargs):
 )
 set_counter("desecapi_pdns_keys_fetched", "number of times pdns keys were fetched")
 
+# pch.py metrics
+set_counter(
+    "desecapi_pch_request_success",
+    "number of times PCH request was successful",
+    ["method", "status"],
+)
+set_counter(
+    "desecapi_pch_request_failure",
+    "number of times PCH request failed",
+    ["method", "path", "status"],
+)
```

Comment:
I think it's ok to keep these separate, so no need to do any work here (just summarizing from offline discussion)